.github/actions/install/action.yml

```diff
@@ -35,12 +35,19 @@ runs:
     - name: Check out repository
       if: inputs.variant == 'dev'
       uses: actions/checkout@v3
+    - name: install current repository
+      run: pip install .
     - name: Install using pip (requirements-dev.txt)
       if: inputs.variant == 'dev'
       shell: bash -l {0}
+      run: pip install --progress-bar off -r requirements-dev.txt
+
+    - name: Check if idaes_ui module exists
       run: |
-        pip install --progress-bar off -r requirements-dev.txt
-        pip install -e .
+        python -c "import idaes_ui; print(idaes_ui)"
+        which python
+        python --version
+        pip list
```

Comment:
Note that changes made to `.github/actions/install/action.yml` will not impact CI runs within the same PR because the workflows are using the action from `main`: https://github.com/IDAES/idaes-ui/blob/4e2c9a208c72c399ac8ab556967b8fff2507fd2f/.github/workflows/core.yml#L69

To debug the `ModuleNotFoundError` that's happening using these commands (which I think is a great approach), I'd suggest to add them directly to the "Start UI" step in `.github/workflows/core.yml` before the `python -m idaes_ui.fv.example` line.